uv.lock

```diff
@@ -591,7 +591,7 @@ wheels = [
 
 [[package]]
 name = "craft-parts"
-version = "2.4.1"
+version = "2.5.0"
```

Comment:
done with `uv lock --upgrade-package craft-parts`
I've always been a big uv fan, it is known